task/black/0.1/black.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: black
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: formatter, python
+    tekton.dev/displayName: "Python Black"
+spec:
+  description: >-
+    This task can be used to format Python code
+  workspaces:
+    - name: shared-workspace
+      description: A workspace that contains the fetched git repository.
```

Comment:
Can we have some better description like: `A workspace containing files that we want to format.`

As workspace may not necessarily contain fetched git repository it may also contain files mounted using `ConfigMap`